Foundation/CGFloat.swift

```diff
@@ -611,6 +611,7 @@ public func %=(lhs: inout CGFloat, rhs: CGFloat) {
     fatalError("%= is not available.")
 }
 
+#if !os(Android)
```

Comment:
IIRC tgmath is available on Android, and I think the apple/swift tests for tgmath pass when run on Android as well. I must be missing something, but why the `#if !os(Android)` here? And do the Foundation tests pass this way?